openlibrary/plugins/upstream/mybooks.py

```diff
@@ -0,0 +1,430 @@
+import json
+import web
+
+from infogami.utils import delegate
+from infogami.utils.view import safeint, render
+
+from openlibrary import accounts
+from openlibrary.core.booknotes import Booknotes
+from openlibrary.core.bookshelves import Bookshelves
+from openlibrary.core.lending import add_availability
+from openlibrary.core.observations import Observations, convert_observation_ids
+from openlibrary.core.sponsorships import get_sponsored_editions
+from openlibrary.plugins.upstream import borrow
+
+
+RESULTS_PER_PAGE = 25
+
+
+class public_my_books(delegate.page):
```

Comment:
```suggestion
class my_books_redirect(delegate.page):
```
`public_my_books` doesn't really explain what is going on here.